### PR TITLE
BCI-1260 Port aggregator proxy tests from hardhat to cairo-test

### PR DESCRIPTION
**Pull request by calvwang9**
https://smartcontract-it.atlassian.net/browse/BCI-1260

### CODE REVIEW

**Review by augustbleeds — APPROVED** [review]
Looks great, appreciated the detailed code comments

File: contracts/src/tests/test_aggregator_proxy.cairo [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    let MockAggregator1 = IMockAggregatorDispatcher { contract_address: MockAggregatorAddr1 };
+
+    // Deploy mock aggregator 2
+    let (MockAggregatorAddr2, _) = deploy_syscall(
+        MockAggregator::TEST_CLASS_HASH.try_into().unwrap(), 0, calldata.span(), false
+    ).unwrap();
+    let MockAggregator2 = IMockAggregatorDispatcher { contract_address: MockAggregatorAddr2 };
+
+    // Return account, mock aggregator address and mock aggregator contract
+    (account, MockAggregatorAddr1, MockAggregator1, MockAggregatorAddr2, MockAggregator2)
```
Comment (cfal) [inline contracts/src/tests/test_aggregator_proxy.cairo]: nit: can variables start with lowercase? eg mockAggregator

**Review by cfal — APPROVED** [review]
looks great!

File: contracts/test/ocr2/aggregator_proxy.test.ts
Comment (cfal) [inline contracts/test/ocr2/aggregator_proxy.test.ts:1]: also - would you mind not deleting this file for now? the reason is that i have it working in the gauntlet deploy PR - it'll save me from some merge conflicts, and we'd be able to keep the commit for the working aggregator_proxy.test.ts (before removal), so it'd be easier to find in git history if we wanted to refer to it later 🙂

File: contracts/src/tests/test_aggregator_proxy.cairo (hunk 1 above)
Comment (calvwang9) [reply contracts/src/tests/test_aggregator_proxy.cairo]: good call, fixed

File: contracts/test/ocr2/aggregator_proxy.test.ts
Comment (calvwang9) [reply contracts/test/ocr2/aggregator_proxy.test.ts:1]: no worries, undid file deletion

**Pull request merged**